libs/pages/application/src/lib/ui/need-redeploy-flag/need-redeploy-flag.tsx

```diff
@@ -0,0 +1,38 @@
+import { ServiceDeploymentStatusEnum } from 'qovery-typescript-axios'
+import { ApplicationEntity } from '@console/shared/interfaces'
+import { Button, ButtonSize, ButtonStyle, IconAwesomeEnum } from '@console/shared/ui'
+
+export interface NeedRedeployFlagProps {
+  application: ApplicationEntity
+  onClickCTA?: () => void
+}
+
+export function NeedRedeployFlag(props: NeedRedeployFlagProps) {
```

Comment:
The name of the component is weird, should be a `Banner` or `BannerLarge`, just start with `Banner` and we need a shared component because we find this on another page, for example here:  

<img width="1118" alt="Capture d’écran 2022-09-06 à 09 12 00" src="https://user-images.githubusercontent.com/533928/188569971-ae0ee773-4d0a-4acb-9cbf-1387f8ba0a84.png">